server/src/utils/fight/fightMethods.ts

```diff
@@ -949,11 +964,18 @@ const activateSuper = (
           // Remove pet from fight
           fightData.fighters = fightData.fighters
             .filter((f) => f.index !== pet.index);
+        } else {
+          // Set remaining pets to play just after
+          pet.initiative = fighter.initiative - 0.01;
         }
       }
 
-      // Abort if no pet feared
-      if (fearSteps.length === 0) return false;
+      // Reduce opponents initiative
+      const opponents = getOpponents({ fightData, fighter, bruteOnly: true });
+      opponents.forEach((opponent) => opponent.initiative -= 0.2 );
```

Comment:
No need to assign them to a const if they are not used afterwards, just .forEach on the getOpponents directly